crates/tabby-common/src/api/event.rs

```diff
@@ -95,35 +93,23 @@ pub struct Segments {
 }
 
 pub trait EventLogger: Send + Sync {
```

Comment:
keep the name `RawEventLogger`

Reply:
Curious to know the rationale? If there is no EventLogger, and it can handle both structs and "raw" events (as strings), what makes the trait specifically a "Raw" event logger?